bazel/private/bazel_cc_proto_library.bzl

```diff
@@ -196,3 +208,46 @@ rules to generate C++ code for.""",
     provides = [CcInfo],
     toolchains = toolchains.use_toolchain(_CC_PROTO_TOOLCHAIN),
 )
+
+def _aspect_hint_impl(ctx):
+    return _ProtoCcAspectHintInfo(copts = ctx.attr.copts)
+
+cc_proto_aspect_hint = rule(
+    implementation = _aspect_hint_impl,
+    attrs = {
+        "copts": attr.string_list(),
```

Comment:
we can add any other attributes we want here, i imagine `linkopts` would be the next most sensible one, happy to add other useful ones in this PR, or just as use cases come up